xml_converter/generators/proto_templates/node.proto

```diff
@@ -0,0 +1,274 @@
+syntax = "proto3";
+import "google/protobuf/any.proto";
+
+package Proto_Node;
+
+message Category {
+    bool default_visibility = 1;
+    string display_name = 2;
+    bool is_separator = 3;
+    string name = 4;
+    string tooltip_name = 5;
+    map <string, Category> children = 6;
+}
+
+message Icon {
+    fixed32 achievement_bit = 1;
+    int32 achievement_id = 2;
+    float alpha = 3;
+    bool can_fade = 4;
+    float distance_fade_end = 5;
+    float distance_fade_start = 6;
+    float height_offset= 7;
+    float __tentative__scale = 8;
```

Comment:
A `__tentative__`, `bhdraft__` or otherwise not finalized value should not be given a single byte index. This should probably be expanded to at least a double byte but probably a triple byte index instead.

Reply:
Rework to be at index 2048 and up.